Anchal_Hora/nodejs/UI/js/delete.js

```diff
@@ -0,0 +1,34 @@
+document.getElementById("buttondelete").addEventListener("click",admindeletedata)
+function admindeletedata(){ 
```

Comment:
when you have multiple words use camelCase

Reply:
Used camelCase Convention